core.lua

```diff
@@ -17,18 +17,33 @@ local function getSeals()
     return seals
 end
 
+local function getEnhancements() 
+    if enhancements then
+        return enhancements
+    end
+    enhancements = {"c_base"}
+    for k, v in pairs(G.P_CENTER_POOLS["Enhanced"]) do
+	    sendDebugMessage("Adding "..v.name.." from G.P_CENTER_POOLS["..k.."]")
```

Comment:
For this, I like to maintain compatibility with needing steamodded (which I just checked and realized I messed up on the seal api for, whoops). As such this causes a crash when steamodded is not loaded. If it could be removed that would be lovely.